cms/articles/tests/test_models.py

```diff
@@ -251,8 +252,125 @@ def test_breadcrumb_doesnt_containt_series_url(self):
-        """Test the page loads in external env."""
+    def test_pagination_is_not_shown(self):
         response = self.client.get(self.basic_page_url)
-        self.assertEqual(response.status_code, 200)
+        expected = 'class="ons-pagination__link"'
+        self.assertNotContains(response, expected)
+
+
+class PreviousReleasesWithoutPaginationTestCase(TestCase):
+    # PREVIOUS_RELEASES_PER_PAGE is default value 10
+    @classmethod
+    def setUpTestData(cls):
+        cls.article_series = ArticleSeriesPageFactory(title="Article Series")
+        cls.articles = StatisticalArticlePageFactory.create_batch(9, parent=cls.article_series)
+        cls.previous_releases_url = cls.article_series.url + cls.article_series.reverse_subpage("previous_releases")
+
+    def test_breadcrumb_does_containt_series_url(self):
+        response = self.client.get(self.previous_releases_url)
+        parent_page = self.article_series.get_parent()
+        # confirm that current breadcrumb is there abd that current breadcrumb points to the parent page
+        # TODO currently this test points to the article page not to the series page
+        self.assertContains(
+            response,
+            f'<a class="ons-breadcrumbs__link" href="{parent_page.url}">{parent_page.title}</a>',
+            html=True,
+        )
+
+    def test_page_content(self):
+        response = self.client.get(self.previous_releases_url)
+        mock_response = response.content.decode("utf-8").split("\n")
+        allowed_to_print = False
+        release_count = 0
+        for m in mock_response:
+            if " </ul>" in m:
+                allowed_to_print = False
+            if "<ul>" in m:
+                allowed_to_print = True
+            if allowed_to_print and "<li><a href=" in m:
+                release_count += 1
+        self.assertEqual(release_count, 9)
+
+    def test_pagination_is_not_shown(self):
+        response = self.client.get(self.previous_releases_url)
+        expected = 'class="ons-pagination__link"'
+        self.assertNotContains(response, expected)
+
+
+@override_settings(PREVIOUS_RELEASES_PER_PAGE=3)
+class PreviousReleasesWithPaginationPagesTestCase(TestCase):
+    total_batch = 13
+    test_previous_releases_per_page = 3
+    total_no_of_pages = math.ceil(total_batch / test_previous_releases_per_page)
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.article_series = ArticleSeriesPageFactory(title="Article Series")
+        cls.articles = StatisticalArticlePageFactory.create_batch(13, parent=cls.article_series)
+        cls.previous_releases_urls = [
+            (cls.article_series.url + cls.article_series.reverse_subpage("previous_releases") + f"?page={3}"),
+            (cls.article_series.url + cls.article_series.reverse_subpage("previous_releases") + f"?page={1}"),
+            (cls.article_series.url + cls.article_series.reverse_subpage("previous_releases") + f"?page={5}"),
+        ]
```

Comment:
```suggestion
        previous_releases_url = cls.article_series.url + cls.article_series.reverse_subpage("previous_releases")
        cls.previous_releases_urls = [
            f"{previous_releases_url}?page=3"),
            f"{previous_releases_url}?page=1"),
            f"{previous_releases_url}?page=5"),
        ]
```

Reply:
Hi Dan like this appeals to my repeated code aversion 
you didn't need the last bracket in the list  lines

Reply:
good eye. updated. the GitHub comment box is not very conducive for editing code snippets